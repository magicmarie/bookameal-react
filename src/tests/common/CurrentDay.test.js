import React from "react";
import { shallow } from "enzyme";
import CurrentDay from "../../components/common/CurrentDay";

describe("AdminDashboard component", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<CurrentDay />);
  });
  it("should render correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
