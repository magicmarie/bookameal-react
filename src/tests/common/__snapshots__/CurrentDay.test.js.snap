// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdminDashboard component should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <getCurrentDay />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): "Thursday",
  Symbol(enzyme.__nodes__): Array [
    "Thursday",
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getSnapshotBeforeUpdate": true,
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
